examples/module1/lesson1/validate-it/index.html

```diff
@@ -11,27 +11,29 @@
       <div class="flex flex-col">
         <input
           class="rounded-md p-4 shadow-lg"
-          type="text"
+          type="number"
+          min="1"
```

Comment:
użyłbym tutaj constantsów które zaimplementowałeś wyżej

Reply:
Jako, że nie używamy w tym zadaniu żadnego frameworku to trochę nie wiem jak się do tego zabrać :sweat_smile:.
Dodałem nowy event listener do inputa i tam to ustawiam. O to chodziło ? :pray:  Wydaje mi się lekki overkill, ale może przy większej aplikacji to słuszne podejście.